Formula/pidgin.rb

```diff
@@ -54,6 +54,11 @@ def install
 
     ENV["ac_cv_func_perl_run"] = "yes" if MacOS.version == :high_sierra
 
+    inreplace "finch/finch.c", "LIBDIR", "\"#{HOMEBREW_PREFIX}/lib/purple-2\""
```

Comment:
Could these be `configure` arguments in future?

Reply:
The problem is the symlinking in the homebrew tree.  The configure argument is `--prefix` and pidgin is looking there by default.  It's only the linking phase that homebrew does that causes this not to work and as such doesn't seem like it should be an upstream fix to me.

For example, `--prefix` is `/usr/local/Cellar/pidgin/2.13.0/` so pidgin/libpurple look in `/usr/local/Cellar/pidgin/2.13.0/lib/purple-2` which works for everything in tree and why I suspect that the otr plugin as added into the main formula as well.

However, formula's shouldn't be installing files to another formula's cellar path, so we have to convince libpurple to ignore it's prefix and instead look in another place (`/usr/local/lib/purple-2`) where other formulas can install files for their accompanying plugins.

Reply:
> as such doesn't seem like it should be an upstream fix to me

I don't have to be but given Homebrew is a pretty widely used downstream I don't trust us (or one of our contributors) to not mess this up in future. The benefit of `configure` arguments is that you, upstream, can decide on how they work rather than us randomly patching files (where we'll be guaranteed to miss some on a version update).

To be clear: this is not blocking but just a "nice to have" I mention as you're upstream.

Reply:
It won't happen upstream.  Pidgin 2 has one more minor release in it before it goes into feature freeze forever.  We have minimal resources and need to point that at our next major revision that's been slowly being worked on for a decade and thus pidgin 2 will be entering maintenance mode at that point.

Reply:
In that case: could this be something that could be included in Pidgin 3? I am suggesting `configure` flags to set `LIBDIR` and `DATADIR` rather than indefinitely patching.